lib/northbound.h

```diff
@@ -538,6 +596,29 @@ enum nb_client {
 	NB_CLIENT_GRPC,
 };
 
+/* Northbound context. */
+struct nb_context {
+	/* Northbound client. */
+	enum nb_client client;
+
+	/* Northbound user (can be NULL). */
+	const void *user;
+
+	/* Client-specific data. */
+#if 0
+	union {
```

Comment:
do we need this code under #if 0?

Reply:
That was necessary to get past the c++ compatibility smoke test, since c++ doesn't allow empty structures and unions :(

If you prefer I can remove the `client_data` field for now and then readd it later only when there's a need for it (e.g. when the PCEP code comes in)